src/main.ts

```diff
@@ -1,16 +1,18 @@
 import * as core from '@actions/core';
-import { ToolsDirectory } from './toolsDirectory';
+
+import * as action from './action';
+import * as cake from './cake';
+import * as cakeTool from './cakeTool';
 import { CakeToolSettings } from './cakeToolSettings';
-import { isError, isString } from './guards';
 import * as dotnet from './dotnet';
-import * as cakeTool from './cakeTool';
-import * as cake from './cake';
-import * as action from './action';
+import { isError, isString } from './guards';
+import { ToolsDirectory } from './toolsDirectory';
```

Comment:
**nitpick:** the order of the `import` statements is deliberate. Please, revert this change.

Reply:
Looks like this was fixed with the rebase